ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -31,7 +31,7 @@ def bufr2ioda(current_cycle, RUN, DMPDIR, config_template_dir, COM_OBS):
 
     # Specify observation types to be processed by a script
     BUFR_py = ["satwind_amv_goes", "satwind_scat", "adpupa_prepbufr", "adpsfc_prepbufr", "sfcshp_prepbufr", "acft_profiles_prepbufr",
-               "gpsro_bufr", "conventional_prepbufr_ps"]
+               "gpsro_bufr", "conventional_prepbufr_ps", "bufr2ioda_subpfl_argo_profiles.py"]
```

Comment:
I believe this script changed so that is no longer needed but that may be only for the current code sprint. @CoryMartin-NOAA what's the status of the line in this file?